src/style/NewTab.less

```diff
@@ -0,0 +1,87 @@
+.tab-user {
+    font-family: "Tajawal", sans-serif;
+    font-weight: 600;
+    .tab-login {
```

Comment:
Please add a comment on why login here is different from our normal widget. Thanks.

Reply:
Outdated, removed